apps/meteor/package.json

```diff
@@ -261,7 +261,7 @@
 		"fibers": "^5.0.1",
 		"file-type": "^10.11.0",
 		"filenamify": "^4.3.0",
-		"filesize": "^3.6.1",
+		"filesize": "^8.0.7",
```

Comment:
oh-no.. this dep again? I thought you have updated it in another PR while ago 🤔

Reply:
The PR that updated it never got merged lol (it's still around, like a ghost)